sds_data_manager/stacks/sds_data_manager_stack.py

```diff
@@ -249,6 +256,10 @@ def __init__(
             runtime=lambda_.Runtime.PYTHON_3_9,
             timeout=cdk.Duration.minutes(15),
             memory_size=1000,
+            allow_public_subnet=True,
```

Comment:
Why would the indexer lambda need to run on a public subnet? Is this for the DB connection/secrets manager?

In the future I think we should look at tightening some of these up to within the private VPC routes if we can. Not necessary right now though.